tools/init-tfvars.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+# fills a terraform vars file with values that are required for the Terraform configuration to work
```

Comment:
idea:
https://developer.hashicorp.com/terraform/language/values/variables#variable-definitions-tfvars-files

maybe use a node script that outputs the terraform vars in json
probably simpler to manage and don't really need to parse the terraform state/providers.

so it can read the `enabled_connectors` variable and generate the missing values if non-existent
so adding a new connector will just be enabling it and run this on top, it may complement existing config